indigo/indigo/src/main/scala/indigo/shared/formats/GenericTileset.scala

```diff
@@ -0,0 +1,39 @@
+package indigo.shared.formats
+
+import indigo._
+
+/** Utility for working with generic sprite sheets. The
+  * sheet must be uniform, with each sprite having the same
+  * width and height, and a (possibly zero-pixel) margin.
+  */
+final case class GenericTileset(
```

Comment:
We could just call it `Tileset`?

Reply:
Sure.

Reply:
Turns out `TileSet` is already being used in TiledMap.scala. I would have thought that `Tileset` with a lowercase s would not conflict, but I get all kinds of scalajs linking errors when I change the name. What should I do?